SwahiliBox/swahilibox-events-api
# Setup database models

#### What does this PR do?
Moves the current existing database models to use Knex.js
Also adds some additional tables as described in this issue #39 

#### Description of Task to be completed?

- setup knex js
- add user and events migration files. 

#### How should this be manually tested?
pull the current branch and run `yarn db:migrate:latest`
this should create all the tables described in the migration files.

#### What are the relevant tasks or issues?
this PR implements issue #39
